src/app/modules/core/components/appcharts/appcharts.component.ts

```diff
@@ -125,14 +125,31 @@ export class AppchartsComponent implements OnInit {
         }
       }],
       xAxes: [{
+        // type: 'time',
+        // time: {
+        //   parser: 'MM/DD/YYYY HH:mm',
+        //   tooltipFormat: 'll HH:mm',
+        //   unit: 'minute',
+        //   unitStepSize: 10,
+        //   displayFormats: {
+        //     second: 'HH:mm:ss',
+        //     minute: 'HH:mm:ss',
+        //     hour: 'HH:mm'
+        //   }
+        // },
+        // ticks: {
+        //   unit: 'minute',
+        //   stepSize: 10,
+        // },
+
```

Comment:
Can we remove this to clean up the code?

Reply:
Removed. Meant to revisit later. Will save a copy for myself for later.